Dockerfile.rhel7

```diff
@@ -60,8 +60,15 @@ ENV NEXUS_DATA=/nexus-data \
     USER_NAME=nexus \
     USER_UID=200
 
-RUN mkdir -p ${NEXUS_HOME} && \
-    curl --fail --silent --location --retry 3 \
+RUN useradd -l -u ${USER_UID} -r -g 0 -m -d ${NEXUS_DATA} -s /sbin/no-login \
+            -c "${USER_NAME} application user" ${USER_NAME} && \
+    mkdir -p ${NEXUS_HOME} && \
+    chown ${USER_NAME} ${NEXUS_HOME}
+
+# Supply non variable to USER command ${USER_NAME}
+USER ${USER_NAME}
```

Comment:
Only reason was trying to keep just one value for being consistent. If Docker linter does not support it, it is ok to use the previous hardcoded value.